web-crypto/sign-verify/index.html

```diff
@@ -17,6 +17,7 @@ <h1>Web Crypto: sign/verify</h1>
           <li>"RSA-PSS"</li>
           <li>"ECDSA"</li>
           <li>"HMAC"</li>
+          <li>"ED25519" (See <a href="https://blogs.igalia.com/jfernandez/2023/06/20/secure-curves-in-the-web-cryptography-api/">Secure Curves in the Web Cryptography API</a>; launch Chrome or Chromium with <a href="https://peter.sh/experiments/chromium-command-line-switches/#enable-experimental-web-platform-features"><code>--enable-experimental-web-platform-features</code></a> command-line switch)</li>
```

Comment:
Bit nitpicky, but the name of the algorithm has a lowercase "d", unlike all the other algorithm names which only include capitals, so there's no need to capitalize it here IMHO.
```suggestion
          <li>"Ed25519" (See <a href="https://blogs.igalia.com/jfernandez/2023/06/20/secure-curves-in-the-web-cryptography-api/">Secure Curves in the Web Cryptography API</a>; launch Chrome or Chromium with <a href="https://peter.sh/experiments/chromium-command-line-switches/#enable-experimental-web-platform-features"><code>--enable-experimental-web-platform-features</code></a> command-line switch)</li>
```

Reply:
Great, thank you. Let's add it 👍🏻